src/plugins/timelist/Timelist.vue

```diff
@@ -357,21 +357,19 @@ export default {
             this.listActivities();
         },
         filterActivities(activity, index) {
+            if (this.isEditing) {
+                return true;
+            }
 
             const hasFilterMatch = this.filterByName(activity.name);
-
             if (hasFilterMatch === false || this.hideAll === true) {
                 return false;
             }
 
-            if (this.showAll === true) {
```

Comment:
Are we using `this.showAll` anymore at all? If not, we should get rid of it.

Reply:
I don't believe we are, but I wasn't sure if I was overlooking a use case for it. Happy to fully remove it.